cedar-policy-formatter/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "cedar-policy-formatter"
-version = "3.2.0"
+version = "3.2.1"
 edition = "2021"
-rust-version = "1.76.0" # minimum supported Rust version is currently 1.76.0 because `cedar-policy-core` requirement. Check with `cargo install cargo-msrv && cargo msrv --min 1.75.0`
+rust-version = "1.76.0"                                         # minimum supported Rust version is currently 1.76.0 because `cedar-policy-core` requirement. Check with `cargo install cargo-msrv && cargo msrv --min 1.75.0`
```

Comment:
probably more whitespace than we need.  (Similar whitespace changes in other Cargo.tomls too)